JSDemos/Demos/Calendar/MultipleSelection/Vue/description.md

```diff
@@ -0,0 +1 @@
+add description
```

Comment:
Why do we have 2 different files for description places in framework dirs? Seems to be incorrect

Reply:
decsription was automaticaly copy pasted from overview demo, I just manually updated it to some text, anyway techwriters will update them